dev.ares.ares.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: dev.ares.ares
+runtime: org.gnome.Platform
```

Comment:
do you really need the  GNOME platform? Isn't the freedesktop SDK enough?

Reply:
I probably could, but I get issues with icons which I am not getting with the GNOME runtime.

Reply:
`Error loading theme icon 'document-open' for stock: Icon 'document-open' not present in theme Adwaita`

Reply:
I am not getting this issue with `bsnes` though which is weird considering both code bases are pretty similar.

Reply:
fair enough.

(not sure how all of this work)

Reply:
I'll ask on our matrix channel just in case